test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs

```diff
@@ -117,53 +134,86 @@ public void Migrating_scripts_with_multiple_commands_creates_Exec_task_for_each(
         [InlineData("postcompile")]
         [InlineData("prepublish")]
         [InlineData("postpublish")]
-        public void Migrated_ScriptSet_has_Exec_and_replaces_variables(string scriptName)
+        public void MigratedScriptSetHasExecAndReplacesVariables(string scriptName)
         {
             var scriptMigrationRule = new MigrateScriptsRule();
             ProjectRootElement mockProj = ProjectRootElement.Create();
 
             var commands = new[] { "%compile:FullTargetFramework%", "%compile:Configuration%"};
 
-            var target = scriptMigrationRule.MigrateScriptSet(mockProj, mockProj.AddPropertyGroup(), commands, scriptName);
+            var target = scriptMigrationRule.MigrateScriptSet(
+                mockProj,
+                mockProj.AddPropertyGroup(),
+                commands,
+                scriptName);
             target.Tasks.Count().Should().Be(commands.Length);
 
             foreach (var task in target.Tasks)
             {
                 var taskCommand = task.GetParameter("Command");
                 var commandIndex = Array.IndexOf(commands, taskCommand);
 
-                commandIndex.Should().Be(-1, "Expected command array elements to be replaced by appropriate msbuild properties");
+                commandIndex.Should().Be(
+                    -1,
+                    "Expected command array elements to be replaced by appropriate msbuild properties");
             }
         }
 
         [Fact]
-        public void Formatting_script_commands_replaces_unknown_variables_with_MSBuild_Property_for_environment_variable_support()
+        public void PublishIISCommandDoesNotGetMigratedBecauseItIsNowInTheWebSDK()
```

Comment:
That's what is being used in other places in the code base and that's what is being used in the SDK repo as well. So, I just started unifying it. Plus, it saves a bit of horizontal space.